frontend/src/components/layout/navigation/whatsnew/WhatsNewDashboard.test.tsx

```diff
@@ -158,12 +158,17 @@ describe("The 'What's new' dashboard", () => {
       />,
     );
 
-    expect(screen.queryByText(allEntries[1].content)).not.toBeInTheDocument();
+    expect(typeof allEntries[1].content).toBe("string");
+    if (typeof allEntries[1].content !== "string") {
+      return;
+    }
```

Comment:
TIL (via Copilot):

> The if (typeof ...) { return; } block is not strictly necessary if the expect(typeof ...).toBe("string") call already ensures that allEntries[1].content is a string. However, the if block provides a safeguard to exit the function early if the type check fails, preventing any further execution that relies on the assumption that allEntries[1].content is a string. This can be useful to avoid runtime errors and make the test more robust.